### DISCUSSION TITLE
Change the visible repository name on GitHub Pages

### DISCUSSION THREAD

**Discussion by Y-Singh** [community/community#23533] [Other Feature Feedback, Questions, & Ideas]
<p>Hi</p>
<p>I’ve been learning a bit about GitHub pages and I wanted to use it to track the milestones for my grad project, but there’s one thing I’ve been wondering about. The title on the page itself is my repository name. I was wondering if there’s anyway to change it without directly changing my repo’s name itself.  I didn’t see anything in the markdown file that would let me change it.</p>
<p>Here is the link to my Githubs page : <a href="https://y-singh.github.io/capstone-project/" rel="noopener nofollow ugc">https://y-singh.github.io/capstone-project/</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/5/c/5c4a00de48d586b07578fab0aa6761cc758b39fd.png" data-download-href="/uploads/short-url/daqtGzuOOYQmIELE4sXrph0cF4x.png?dl=1" title="Screenshot 2022-02-03 100022" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47701821/181077756-1db5c17e-c148-4386-bd3f-4ce836b1307d.png" alt="Screenshot 2022-02-03 100022" data-base62-sha1="daqtGzuOOYQmIELE4sXrph0cF4x" width="690" height="140" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/c/5c4a00de48d586b07578fab0aa6761cc758b39fd_2_690x140.png, https://aws1.discourse-cdn.com/github/optimized/3X/5/c/5c4a00de48d586b07578fab0aa6761cc758b39fd_2_1035x210.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/5/c/5c4a00de48d586b07578fab0aa6761cc758b39fd.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/c/5c4a00de48d586b07578fab0aa6761cc758b39fd_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screenshot 2022-02-03 100022</span><span class="informations">1361×278 20.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by yoannchaudet** [comment]
<p>Hello and welcome here!</p>
<p>You can configure that very easily by adding a Jekyll configuration file in your repository (a <code>_config.yml</code>) at the root of your repository with the following content:</p>
<pre><code class="lang-auto">title: the title you want
</code></pre>
<p>This should have the same effect than renaming your repository (like <a href="https://github.community/t/bug-report-wrong-domain-shown-at-the-top-of-the-page/225027/4">here</a>).</p>
<p>Hope this helps!</p>

**Comment by Y-Singh** [comment]
<p>Thanks so much, it works! Is there a list of other things that can be changed by modifying the config file?</p>

**Comment by yoannchaudet** [comment]
<p>Many things but I will need to refer you to the Jekyll documentation: <a href="https://jekyllrb.com/docs/" class="inline-onebox">Quickstart | Jekyll • Simple, blog-aware, static sites</a></p>
<p>Jekyll is a static site generator that GitHub Pages is using by default.</p>

**Comment by Y-Singh** [comment]
<p>Thank you! I’ll refer to this first in case I have any other questions.</p>
